documentation/staging/content/managing-domains/domain-lifecycle/retry.md

```diff
@@ -0,0 +1,113 @@
+| Internal             | The operator encountered an internal Exception while processing the Domain resource.                                                                                                               |
+| TopologyMismatch     | One or more servers or clusters configured in the domain resource do not exist in the WebLogic domain configuration, or monitoring exporter port is specified and it conflicts with a server port. |
+| Aborted              | The introspector encountered a FATAL error, or the operator has exceeded the maximum retry time.                                                                                                   |
+
+### Domain Failure Severities
+
+Domain resource failures fall into the 3 different severities:
+
+1. Fatal failures
+   - Failures that any further retries will also fail unless the cause of the failure if fixed. \
+     Examples:
+     1) FATAL errors in the introspector log that contains the special marker string `FatalIntrospectorError`
+     2) Validation errors in domain resource that requires changes to the domain spec.
+     3) Failures in the Severe category have reached the expected maximum retry time.
+2. Severe failures
+   - Most of the failures during domain processing are temporary failures that may either resolve in a later 
+     time without user intervention, or be fixed by the user's actions without making any change to the domain resource. \
+     Examples:
+     1) introspector job time out (DeadlineExceeded error)
+     2) SEVERE errors in the introspector log that does not contain the special marker string `FatalIntrospectorError`
+     3) temporary network issues
+     4) unauthorized to create some resources
+     5) Exception from the operator itself
+     6) Validation errors that can be fixed without changing the domain spec, for example, missing secrets, or missing configmap.
+3. Warnings
+   - Mismatch in domain spec configuration and WebLogic domain topology that usually do not prevent the domain from becoming available. 
+     For example, Replicas are configured too high
+     
+{{%expand "Click here to view an example of domain status showing a failure with severity and reason." %}}
+```yaml
+Status:
+   ...
+  Conditions:
+    Last Transition Time:  2022-10-10T23:48:09.157398Z
+    Message:               10 replicas specified for cluster 'cluster-1' which has a maximum cluster size of 5
+  10 replicas specified for cluster 'cluster-2' which has a maximum cluster size of 2
+Reason:                ReplicasTooHigh
+Severity:              Warning
+Status:                True
+Type:                  Failed
+   ...
+```
+{{% /expand %}}
+
+### Retry Behavior
+
+Domains that have failures with severity of `Fatal` or `Warning` will not be retried. The domain status should contain a message indicating what action is needed to fix the failure condition. 
+
+Domains that have severe failure conditions will be retried as follows:
```

Comment:
(to match the previous paragraph)

Domains that have severe failure conditions  -> Domain failures with a severity of `Severe`